src/components/AdditionalFeature.js

```diff
@@ -1,13 +1,33 @@
 import React from 'react';
+import { addFeature, removeFeature } from '../actions/actions';
+
+import { connect } from 'react-redux';
+const mapStateToProps = state => {
+    return {
+        additionalPrice: state.additionalPrice,
+        car: state.car,
+        additionalFeatures: state.additionalFeatures,
+    };
+}
 
 const AdditionalFeature = props => {
+    let addFeature = event => {
+        event.preventDefault();
+        props.addFeature(props.feature.price);
+    };
+    let removeFeature = event => {
```

Comment:
I would think about how the user interacts with the add. The way it's currently set up it's like the user could remove an additional feature from the options they have of additional features.